### DISCUSSION TITLE
Don't you guys think a DB would be much better than files? I personally think reading files is way expensive and slow. What do you guys think?

### DISCUSSION THREAD

**Discussion by SyedMSawaid** [UdaraJay/Pile#54] [Q&A] [answered]
@UdaraJay Would love you hear your take on this.

**Comment by GTruss** [comment, marked as answer]
I personally prefer the openness of plain-text (markdown) files for notes & journaling. If this were a heavy, data-intensive application I'd agree that a database would be better, but I don't want to be tied into a proprietary database just for this stuff. I switched from a database-driven notetaking app to Obsidian (plain-text markdown files) over a year ago and I'll never go back.

**Reply by SyedMSawaid** [reply]
I agree. But you own your `sqlite` DB and we can have a method/function/feature to export it all to markdown. Win for both.
